lightkurve/lightcurve.py

```diff
@@ -1048,6 +1048,12 @@ def to_periodogram(self, method="lombscargle", **kwargs):
             Keyword arguments passed to either
             `~lightkurve.periodogram.LombScarglePeriodogram` or
             `~lightkurve.periodogram.BoxLeastSquaresPeriodogram`.
+            Keywords accepted by `~lightkurve.periodogram.LombScarglePeriodogram` are:
+            ``min_frequency``, ``max_frequency``, ``min_period``, ``max_period``, ``frequency``,
+            ``normalization``.
+            ``period``, ``nterms``, ``nyquist_factor``, ``oversample_factor``, ``freq_unit``.
+            Keywords accepted by `~lightkurve.periodogram.BoxLeastSquaresPeriodogram` are:
+            ``min_period``, ``max_period``, ``period``, ``frequency_factor``.
```

Comment:
Minor change request: we moved the listing of these options to lines 1031-1038 in this same docstring (this changed just a few weeks ago, i.e. after this PR started).  To avoid repetition, would you be willing to add `normalization` on line 1034 in this docstring and removing lines 1051-1056?

Reply:
Done! I've also removed the line regarding how LombScargle(normalization) is fixed to PSD (technically unnormalized) to avoid confusion with the normalization parameter in to_periodogram().